specification/authorization/resource-manager/Microsoft.Authorization/preview/2021-11-16-preview/authorization-AccessReviewCalls.json

```diff
@@ -0,0 +1,2724 @@
+        }
+      },
+      "description": "Access Review History Instances."
+    },
+    "AccessReviewHistoryInstance": {
+      "x-ms-azure-resource": true,
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The access review history definition instance id."
+        },
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The access review history definition instance unique id."
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The resource type."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/AccessReviewHistoryInstanceProperties",
+          "description": "Access Review History Definition Instance properties."
+        }
+      },
+      "description": "Access Review History Definition Instance."
+    },
+    "AccessReviewHistoryInstanceProperties": {
+      "type": "object",
+      "properties": {
+        "reviewHistoryPeriodStartDateTime": {
+          "type": "string",
+          "format": "date-time",
+          "readOnly": false,
+          "description": "Date time used when selecting review data, all reviews included in data start on or after this date. For use only with one-time/non-recurring reports."
+        },
+        "reviewHistoryPeriodEndDateTime": {
+          "type": "string",
+          "format": "date-time",
+          "readOnly": false,
+          "description": "Date time used when selecting review data, all reviews included in data end on or before this date. For use only with one-time/non-recurring reports."
+        },
+        "displayName": {
+          "type": "string",
+          "description": "The display name for the parent history definition."
+        },
+        "status": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Status of the requested review history instance data. This is either requested, in-progress, done or error. The state transitions are as follows - Requested -> InProgress -> Done -> Expired",
+          "enum": [
+            "Requested",
+            "InProgress",
+            "Done",
+            "Error"
+          ],
+          "x-ms-enum": {
+            "name": "AccessReviewHistoryDefinitionStatus",
+            "modelAsString": true
+          }
+        },
+        "runDateTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Date time when the history data report is scheduled to be generated.",
+          "x-nullable": true
+        },
+        "fulfilledDateTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Date time when the history data report is scheduled to be generated.",
+          "x-nullable": true
+        },
+        "downloadUri": {
```

Comment:
>downloadUri [](http://example.com/codeflow?start=9&length=11)

according to our offline chat, this value is only generated via a POST API. Since GET and POST use the same definition: `AccessReviewHistoryInstanceProperties` recommend marking this value a readonly: true and remove it from your Get**** example.